### PR TITLE
feat: add TypeExpression to BuiltinStmt and fix tests

### PR DESCRIPTION
**Pull request by yuktarakesh**
### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

**Comment by ChitlangeSahas** [comment]
If it helps: https://github.com/influxdata/flux/pull/3083, this is my progress. If it is confusing, don't worry about it. Just wanted to make sure you were aware of that PR.

File: libflux/go/libflux/parser_test.go [hunk 1]
```diff
@@ -248,13 +246,16 @@ re =~ /foo/
 re !~ /foo/
 `,
 		},
+		{
+			name:     "agg_test",
+			fluxFile: ``,
+		},
```
Comment (jlapacik) [inline libflux/go/libflux/parser_test.go:252]: This can be removed. I think this was just for debugging.

File: libflux/go/libflux/parser_test.go [hunk 2]
```diff
@@ -281,21 +282,21 @@ re !~ /foo/
...
+			//	},
+			//}
+			//goJSON, err = json.Marshal(&goAST)
+			//if err != nil {
+			//	t.Fatal(err)
+			//}
+			//goJSON = mustIndent(t, goJSON)
+			//if _, err := libflux.ParseJSON(goJSON); err != nil {
+			//	t.Fatal(err)
+			//}
```
Comment (jlapacik) [inline libflux/go/libflux/parser_test.go:299]: This commented out code can be deleted.

File: libflux/src/core/parser/tests.rs [hunk 3]
```diff
@@ -1300,12 +1300,45 @@ fn builtin() {
...
+// fn builtin_csv() {
+//     let src = r#"
+//
+//     "#;
+//     let mut p = Parser::new(src);
+//     let f = p.parse_file("".to_string());
+//     let m = &mut Fresher::new();
+//     // println!("{}", convert_to_string(&f).unwrap());
+// }
+
```
Comment (jlapacik) [inline libflux/src/core/parser/tests.rs:1341]: I think this was also just debug code.

File: libflux/src/core/semantic/builtins.rs [hunk 4]
```diff
@@ -50,6 +50,7 @@ pub fn builtins() -> Builtins<'static> {
                 "from" => "(?csv: string, ?file: string) => [A] where A: Record",
             },
             "date" => semantic_map! {
+                 "from" => "(?csv: string, ?file: string) => [A] where A: Record",
```
Comment (jlapacik) [inline libflux/src/core/semantic/builtins.rs:53]: Can you remove this line too?

File: libflux/src/core/semantic/convert.rs [hunk 5]
```diff
@@ -711,6 +712,22 @@ mod tests {
...
+
+builtin from : (driverName: string, dataSourceName: string, query: string) => [A]
+builtin to : (<-tables: [A], driverName: string, dataSourceName: string, table: string, ?batchSize: int) => [A]
+        "#;
+        let mut p = parser::Parser::new(src);
+        let f = p.parse_file("".to_string());
+        let m = &mut Fresher::default();
+        convert_file(f, m).unwrap();
+}
+
```
Comment (jlapacik) [inline libflux/src/core/semantic/convert.rs:730]: More debug code.

**Pull request merged**